src/main/psp.js

```diff
@@ -202,10 +300,19 @@ gpii.app.psp.show = function (psp) {
  * changes are `bounds`, `workArea`, `scaleFactor` and `rotation`
  */
 gpii.app.psp.handleDisplayMetricsChange = function (psp, event, display, changedMetrics) {
-    if (changedMetrics.indexOf("workArea") > -1) {
-        var windowSize = psp.pspWindow.getSize(),
-            contentHeight = windowSize[1];
-        psp.resize(contentHeight, true);
+    // In older versions of Electron (e.g. 1.4.1) whenever the DPI was changed, one
+    // `display-metrics-changed` event was fired. In newer versions (e.g. 1.8.1) the
+    // `display-metrics-changed` event is fired multiple times. The change of the DPI
+    // appears to be applied at different times on different machines. On some as soon
+    // as the first `display-metrics-changed` event is fired, the DPI changes are
+    // applied. On others, this is not the case until the event is fired again. That is
+    // why the resizing should happen only the second (or third) time the
+    // `display-metrics-changed` event is fired in which case the changedMetrics argument
+    // will not include the `scaleFactor` string.
+    if (changedMetrics.indexOf("scaleFactor") === -1) {
```

Comment:
I think all nodes are sufficiently modern that we can now use the "includes" method https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes